internal/pkg/bulk/engine.go

```diff
@@ -65,18 +68,28 @@ type Bulk interface {
 	// Accessor used to talk to elastic search direcly bypassing bulk engine
 	Client() *elasticsearch.Client
 
+	CheckRemoteOutputChanged(name string, newCfg map[string]interface{})
```

Comment:
this function is currently only used within bulker and tests, could be made private